livre/eq/render/RayCastRenderer.cpp

```diff
@@ -203,6 +206,9 @@ struct RayCastRenderer::Impl
         tParamNameGL = glGetUniformLocation( program, "nSamplesPerRay" );
         glUniform1i( tParamNameGL, _computedSamplesPerRay );
 
+        tParamNameGL = glGetUniformLocation( program, "nDefaultSamplesPerRay" );
+        glUniform1i( tParamNameGL, nDefaultSamplesPerRay );
+
```

Comment:
Since this can never change at runtime, is there a better way then an uniform to sync this value with the shader?